src/Exception/LicenseException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Shopware\ServiceBundle\Exception;
+
+class LicenseException extends \Exception
```

Comment:
It would be great to have a unique error code for each type of license error. This would be useful for catching exceptions and handling them in different ways.